WordPress/src/main/java/org/wordpress/android/ui/uploads/MediaUploadHandler.java

```diff
@@ -141,8 +146,22 @@ static boolean hasPendingOrInProgressMediaUploadsForPost(PostModel postModel) {
      * it's assumed to be a completed upload (returned progress is 1).
      */
     static float getProgressForMedia(MediaModel media) {
-        Float progress = sProgressByMediaId.get(media.getId());
-        return progress == null ? 0 : progress;
+        if (!sProgressByMediaId.containsKey(media.getId())) {
+            return 1;
+        }
+
+        float uploadProgress = sProgressByMediaId.get(media.getId());
```

Comment:
Ok I know what was [happening with the progress bar](https://github.com/wordpress-mobile/WordPress-Android/pull/6473#issuecomment-320320875): the problem can be solved by reverting this part to how it was before.

Remove this:
```
        if (!sProgressByMediaId.containsKey(media.getId())) {
            return 1;
        }
        float uploadProgress = sProgressByMediaId.get(media.getId());
```
and replace that with this:
```
        Float progress = sProgressByMediaId.get(media.getId());
        float uploadProgress = progress == null ? 0 : progress;
```

I realised the [method description is misleading,](https://github.com/wordpress-mobile/WordPress-Android/pull/6473/files#diff-fd1074f533e624627941f15b88dd3277R146) so we’d better change / eliminate that comment as well.